tests/test.sh

```diff
@@ -174,7 +183,45 @@ go run ./construction-signing
 printf "${GRN}### Testing construction transaction types...${OFF}\n"
 go run ./construction-txtypes
 
-# Clean up after a successful run.
+# Now test if the initial block height change works on a new network.
+printf "${GRN}### Terminating existing test network...${OFF}\n"
+cleanup
 rm -rf "${TEST_BASE_DIR}"
+mkdir -p "${TEST_BASE_DIR}"
+
+printf "${GRN}### Starting new test network at non-1 height...${OFF}\n"
+start_network 123
+NONCE=0
+
+sleep 2
+
+advance_epoch 1
+wait_for_nodes
+
+printf "${GRN}### Starting the Rosetta gateway (again)...${OFF}\n"
+${OASIS_ROSETTA_GW} &
+
+sleep 3
+
+printf "${GRN}### Transferring tokens (1+)...${OFF}\n"
+gen_transfer "${TEST_BASE_DIR}/tx1.json" 1000 "${DST}"
+submit_tx "${TEST_BASE_DIR}/tx1.json"
+
+advance_epoch 2
+wait_for_nodes
+
+advance_epoch 3
+wait_for_nodes
+
+advance_epoch 4
+wait_for_nodes
+
+
+printf "${GRN}### Validating Rosetta gateway implementation (again)...${OFF}\n"
+go run ./check-prep
+./rosetta-cli --configuration-file rosetta-cli-config.json check:data --start 123 --end 135
```

Comment:
What happens if you don't pass `--start`?

Reply:
It still works fine :)
I've removed it, since it's not necessary.